aiohttp/streams.py

```diff
@@ -698,8 +696,23 @@ def feed_data(self, data: _SizedT) -> None:
             self._protocol.pause_reading()
 
     async def read(self) -> _SizedT:
-        try:
-            return await super().read()
```

Comment:
This super call is more expensive than I would expect.

... writing this out twice is not good though....

need to come up with something better